acceptance/tests/resource/file/should_create_symlink.rb

```diff
@@ -2,10 +2,10 @@
 
 message = 'hello world'
 agents.each do |agent|
-  confine_block :to, :platform => 'windows' do
+  if agent.platform.variant == 'windows'
```

Comment:
Just found http://www.rubydoc.info/github/puppetlabs/beaker/Beaker/Platform#variant-instance_method which I think I knew about at some point and forgot